pkg/entities/discord.go

```diff
@@ -336,7 +336,7 @@ func (e *Entity) EditGuildChannel(guildID string, statChannel model.DiscordGuild
 	// }
 
 	newChannelName := util.CreateChannelName(guildStat, statChannel.CountType)
-	_, err = e.discord.ChannelEdit(statChannel.ChannelID, newChannelName)
+	_, err = e.discord.ChannelEdit(statChannel.ChannelID, &discordgo.ChannelEdit{Name: newChannelName})
```

Comment:
is this because new version discordgo ?

Reply:
yes, they changed `string -> *ChannelEdit`